submissions/mod-comparison/mod-comparison.md

```diff
@@ -0,0 +1,106 @@
+
+### Vanilla
+
+The original game is, of course, the baseline. It’s where we (probably!) all started and where everything else has grown from. Most of the recipes are quite simple, in that they generally only have a single output – no byproducts to worry about – and there’s usually only one way to make anything you might need. The game starts off very simple with red science packs taking a mere 4 steps – smelting the iron, smelting the copper, making gears and making the pack – but gradually building up. Blue science has traditionally been the great filter – the point at which a number of players find the game has become difficult, due to having to deal with oil, and then it gets significantly harder again with yellow science, largely due to the complexities of making blue circuits. Or rather, the challenge of making green circuits fast enough to keep up with the requirements for blue!
+
+Of course, I’m sure that anyone reading this blog is well past seeing those things as challenges, but I remember back in the olden days when I first started playing, these things certainly seemed difficult!
+
+To the more experienced player, launching the first rocket is merely the early game. Once all the tech is unlocked, it’s time to start building up massive rail networks for moving resources, towns and city blocks for making intermediate components and ranks of rocket silos for getting the space science flowing in. At this point, the challenge of the game is largely in throughput. You need to deal with enormous quantities of the raw materials, processing them en masse to keep the ravenous factory satisfied and compacting them all down into those oh-so-valuable research packs to feed into the science labs. There is no absolute win condition here, it’s just making your factory work faster and faster. 
+
+The biggest challenges in this sort of factory are in the logistics - making sure your rail networks don’t clog, that your smelteries don’t run out of ore, that your design is kind enough to your computer that the game doesn’t grind to a halt. And these are certainly valid challenges and a lot of thought has been put into making this sort of factory run effectively, however I wanted something a bit different, so I started looking at mods. 
+
+### AngelBobs
+
+AngelBobs refers to all of the mods released by a pair of authors – [Arch666Angel](https://mods.factorio.com/user/Arch666Angel) and [Bobingabout](https://mods.factorio.com/user/Bobingabout). In a nutshell, Angel’s mods add extra complexities to recipes, requiring extra steps like crushing ores, or adding to chemical processing systems, whereas Bob’s mods tend to add in extra equipment to help you deal with it, such as faster belts or inserters, or more dangerous biters and bigger guns to help you deal with them. This isn’t 100% accurate, of course, but was the impression I got. 
```

Comment:
Bobs also adds all the different types of metals, but I don't think you need to go into too much detail